src/D2L.Bmx/D2L.Bmx.csproj

```diff
@@ -1,31 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <AssemblyName>bmx</AssemblyName>
     <PublishAot>true</PublishAot>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
     <InvariantGlobalization>true</InvariantGlobalization>
     <RootNamespace>D2L.Bmx</RootNamespace>
   </PropertyGroup>
-
   <ItemGroup>
     <PackageReference Include="AWSSDK.SecurityToken" Version="3.7.101.37" />
     <PackageReference Include="HtmlAgilityPack" Version="1.11.46" />
-    <PackageReference Include="Microsoft.DotNet.ILCompiler" Version="8.0.0-preview.4.23259.5" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Ini" Version="7.0.0" />
-    <PackageReference Include="Microsoft.Extensions.FileProviders.Physical" Version="7.0.0" />
-    <PackageReference Include="runtime.linux-x64.Microsoft.DotNet.ILCompiler" Version="8.0.0-preview.4.23259.5" />
-    <PackageReference Include="runtime.osx-x64.Microsoft.DotNet.ILCompiler" Version="8.0.0-preview.4.23259.5" />
-    <PackageReference Include="runtime.win-x64.Microsoft.DotNet.ILCompiler" Version="8.0.0-preview.4.23259.5" />
     <PackageReference Include="System.CommandLine" Version="2.0.0-beta4.22272.1" />
+    <PackageReference Include="Microsoft.DotNet.ILCompiler" Version="8.0.0-preview.5.23280.8" />
```

Comment:
including `Microsoft.DotNet.ILCompiler` is actually discouraged. the SDK will set it up as long as we have `<PublishAot>`
```suggestion
```

Reply:
Oh neat, did not know this